actions/insights/advisor_actions.py

```diff
@@ -309,6 +311,8 @@ async def process_openshift_advisor(
             )
         elif category == "cluster":
             return await self.openshift_clusters(dispatcher, tracker, domain, events)
+        elif category == "workload":
+            return await self.openshift_workload(dispatcher, tracker, domain, events)
```

Comment:
Minor thing, should we use those constants here?

```suggestion
        elif category == OPENSHIFT_CATEGORY_CLUSTER:
            return await self.openshift_clusters(dispatcher, tracker, domain, events)
        elif category == OPENSHIFT_CATEGORY_WORKLOAD:
            return await self.openshift_workload(dispatcher, tracker, domain, events)
```

Reply:
Makes sense to me! thanks!